packages/hooks/src/smokeTest/koa.test.ts

```diff
@@ -33,7 +33,7 @@ const run = ({ skipHook, smokeTest, userAgent }: Options) => {
 it.each`
   skipHook | userAgent
   ${true}  | ${'Mozilla/5.0'}
-  ${false} | ${'aws-codedeploy-hook-BeforeAllowTraffic/1.2.3'}
```

Comment:
I think there's some benefit in having the user-agent match the actual function name, which I believe is `aws-codedeploy-hook-BeforeAllowTraffic` in this case. Could we switch this around to fix the user agent rather than the check?